angular/projects/spark-angular/src/lib/components/sprk-stack/sprk-stack.component.spec.ts

```diff
@@ -30,61 +30,103 @@ describe('SprkStackComponent', () => {
   it('should set the correct class for itemSpacing', () => {
     component.itemSpacing = 'tiny';
     fixture.detectChanges();
-    expect(element.classList.contains('sprk-o-Stack--tiny')).toEqual(true);
+    expect(element.classList.toString()).toBe(
+      'sprk-o-Stack sprk-o-Stack--tiny',
+    );
   });
 
   it('should set the correct class for itemSpacing', () => {
     component.itemSpacing = 'small';
     fixture.detectChanges();
-    expect(element.classList.contains('sprk-o-Stack--small')).toEqual(true);
+    expect(element.classList.toString()).toBe(
+      'sprk-o-Stack sprk-o-Stack--small',
+    );
+  });
+
+  it('should add the stack item class if isStackItem is true', () => {
+    component.isStackItem = true;
```

Comment:
I added a new input to stack for isStackItem so that you could have a nested stack.